neutron_vpnaas/services/vpn/device_drivers/ovn_ipsec.py

```diff
@@ -33,6 +35,19 @@
 
 LOG = logging.getLogger(__name__)
 
+meter_opts = [
```

Comment:
can we move this section to meter_manager.py, then it is not necessary to add this section in each driver.